cohttp-lwt-unix/src/debug.ml

```diff
@@ -28,7 +28,13 @@ let default_reporter () =
     let k _ =
       let write () = Lwt_io.write Lwt_io.stderr (fmtr_flush ()) in
       let unblock () = over (); Lwt.return_unit in
-      Lwt.finalize write unblock |> Lwt.ignore_result;
```

Comment:
Can Lwt.finalize actually raise?

Reply:
Yes, IIUC it would forward the exception of both functions:

> finalize f g runs f (), and then g (), whether f () fails or completes normally. Lwt.finalize returns the 
> result of f (), unless g () fails, in which case it returns the exception raised by g ().
(https://ocsigen.org/lwt/dev/api/Lwt)

Reply:
Gotcha